modules/ROOT/pages/_partials/security/security-api-schema.adoc

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
Ping @ziccardi  for quick review. This is just a security schema. 
We have it done thru js-sdk api docs already.